Others/SJF.java

```diff
@@ -0,0 +1,168 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.*;
+
```

Comment:
Hello, please add some descriptions about what the algorithm does. Thx

Reply:
Use Java doc comments, please. See the following example:
```
/**
 * Shortest job first.
 *
 * <p>Shortest job first (SJF) or shortest job next, 
 * is a scheduling policy that selects the waiting process 
 * with the smallest execution time to execute next..........
 *
 * @author shivg7706
 * @since 2018/10/27
 */
```